samcli/lib/telemetry/event.py

```diff
@@ -62,10 +65,20 @@ def __eq__(self, other):
         return self.event_name == other.event_name and self.event_value == other.event_value
 
     def __repr__(self):
-        return f"Event(event_name={self.event_name.value}, event_value={self.event_value})"
+        return (
+            f"Event(event_name={self.event_name.value}, "
+            f"event_value={self.event_value}, "
+            f"thread_id={self.thread_id}, "
+            f"timestamp={self.timestamp})"
+        )
 
     def to_json(self):
-        return {"event_name": self.event_name.value, "event_value": self.event_value}
+        return {
+            "event_name": self.event_name.value,
+            "event_value": self.event_value,
+            "thread_id": self.thread_id,
+            "timestamp": str(self.timestamp)[:-3],
```

Comment:
I'm assuming trimming the last few characters is to make the timestamp format SQL compatible but can we drop a comment somewhere explaining this?